src/shared/proto/BUILD

```diff
@@ -8,7 +8,7 @@ proto_library(
         "robot_component_status.proto",
         "tbots_robot_msg.proto",
     ],
-    visibility = ["//visibility:private"],
+    # Note: this library is public so that proto libraries in firmware or software can build from it
```

Comment:
Can they not use the `tbots_cc_proto`?

Reply:
Sadly no see `src/software/proto/BUILD` can't use `tbots_cc_proto` because the rule expects a `proto_library`

Reply:
:cry: is `tbots_cc_proto` needed then? Anything still use it? (just double checking)

Reply:
I think it's needed. The .h and .c/cpp files that send and receive TbotsRobotMsg will need tbots_cc_proto to build